wazuh/wazuh-kibana-app
# Add Miscellaneous to tab into Configuration App

### Description

To show Miscellaneous into App, we decided to include into Configuration App as a Tab.
 
### Issues Resolved

https://github.com/wazuh/wazuh-kibana-app/issues/5826

### Evidence

![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/8cc7d4a5-0245-409c-a15b-2f75e0d48932)
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/9d650f1c-dca4-479b-9b38-869a97de955e)


### Test

- Set up environment

- Check if these apps tab doesn't render
  -  API
  - Modules
  - Sample Data
  - Logs
  - About

- Check if Miscellaneous app doesn't exits
- Check if Configuration app render as expected
 
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/bb86bd6c-ae49-4740-be2e-9b0d231112ef)



### Check List
